index.js

```diff
@@ -3,63 +3,55 @@ module.exports = {
   extends: "eslint:recommended",
   plugins: ["flowtype", "babel"],
```

Comment:
Seems it is unused as of 3b0fec2ab84af288934f97b9f12cf825b484f063 @hzoo ?
So probably we can remove it from here and from all projects using the config.